.github/workflows/build_main_documentation.yml

```diff
@@ -13,6 +13,10 @@ on:
         type: string
       notebook_folder:
         type: string
+      # supply --not_python_module for HF Course
+      module_flag:
```

Comment:
I'd call this `additional_args` so we can use it to pass anything.

Reply:
I see, https://github.com/huggingface/doc-builder/pull/153/commits/35a58fa1ee0fd6776e32f2519ec83dcda72a760d